src-java/kilda-model/src/main/java/org/openkilda/adapter/FlowDestAdapter.java

```diff
@@ -36,11 +41,6 @@ public FlowEndpoint getEndpoint() {
                 trackConnectedDevices.isDstArp() || trackConnectedDevices.isDstSwitchArp());
     }
 
-    @Override
-    public boolean isMultiTableSegment() {
```

Comment:
I would like to keep this getter (with replaced body), you have all required objects to extract it.

Reply:
Reworked.